criteo_distributed_training/run.sh

```diff
@@ -7,10 +7,14 @@ run_script() {
   MODEL_SIZE=$3
 
   # Display the current configuration
-  echo "Running with ${NUM_NODES} nodes, embedding dimension ${EMBEDDING_DIMENSION}, and ${MODEL_SIZE} model size" 2>&1 | tee -a log_file.txt
+  echo "Running with ${NUM_NODES} nodes, embedding dimension ${EMBEDDING_DIMENSION}, and ${MODEL_SIZE} model size" 
 
   # Execute the script with the current configuration
-  python3 train.py --num_nodes "${NUM_NODES}" --embedding_dimension "${EMBEDDING_DIMENSION}"
+  python3 train.py --num_nodes "${NUM_NODES}" --embedding_dimension "${EMBEDDING_DIMENSION}" 2>&1 | tee -a log_file.txt
+
+  # Clearing previous training results to free disk space for next iteration
+  # If you want to use the trained model, you can save it somewhere else or find it on s3 storage specified in Ray Runtime.
+  rm -rf ~/ray_results train* test*
```

Comment:
I could recommend saving the trained model locally on the head node. I don't think having a more big memory should be a big issue since these models are small.

Reply:
Changed. 
The shell script doesn't delete model now. But still, we would need to delete the `ray_results` folder since that would store train files and that would take too much space.